compose/snippets/src/main/java/com/example/compose/snippets/animations/AnimationSnippets.kt

```diff
@@ -709,6 +723,104 @@ private fun AnimationSpecKeyframe() {
     // [END android_compose_animations_spec_keyframe]
 }
 
+@OptIn(ExperimentalAnimationSpecApi::class)
+@Composable
+private fun AnimationSpecKeyframeWithSpline() {
+    // [START android_compose_animation_spec_keyframes_with_spline]
+    val offset by animateOffsetAsState(
+        targetValue = Offset(300f, 300f),
+        animationSpec = keyframesWithSpline {
+            durationMillis = 6000
+            Offset(0f, 0f) at 0
+            Offset(150f, 200f) atFraction 0.5f
+            Offset(0f, 100f) atFraction 0.7f
+        }
+    )
+    // [END android_compose_animation_spec_keyframes_with_spline]
+}
+
+@Suppress("PrimitiveInCollection")
```

Comment:
Why is this suppression needed? I commented it out but didn't get any compiler warnings